src/components/NewClusterModal.js

```diff
@@ -502,179 +817,366 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
-                ') '
+          ]),
+          renderBottomButtons()
+        ]) :
+        h(Fragment, [
+          div({ style: { padding: '1.5rem', borderBottom: `1px solid ${colors.dark(0.4)}` } }, [
+            renderTitleAndTagline(),
+            renderCostBreakdown()
+          ]),
+          div({ style: { padding: '1.5rem', overflowY: 'auto', flex: 'auto' } }, [
+            renderApplicationSection(),
+            renderRuntimeSection(),
+            !!isPersistentDisk && renderPersistentDiskSection(),
+            !sparkMode && !isPersistentDisk && div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+              div([
+                'Time to upgrade your cloud environment. Terra’s new persistent disk feature will safegard your work and data. ',
+                h(Link, { onClick: () => handleLearnMoreAboutPersistentDisk() }, ['Learn more'])
               ]),
-              div({ style: { margin: '0.5rem 0' } }, ['2. Copy the URL from the github repository']),
-              div({ style: { margin: '0.5rem 0' } }, ['3. Enter the URL for the image in the text box below']),
-              h(ValidatedInput, {
-                inputProps: {
-                  placeholder: 'Paste image path here',
-                  value: customEnvImage,
-                  onChange: customEnvImage => this.setState({ customEnvImage })
-                },
-                error: Utils.summarizeErrors(customEnvImage && errors?.customEnvImage)
-              })
-            ])
-          }],
-          [Utils.DEFAULT, () => {
-            return h(Fragment, [
-              div({ style: { display: 'flex' } }, [
-                h(Link, { onClick: () => this.setState({ viewMode: 'packages' }) }, ['What’s installed on this environment?']),
-                makeImageInfo({ marginLeft: 'auto' })
-              ])
-            ])
-          }]),
-        runtimeConfig(),
-        bottomButtons()
-      ])]
-    )
+              h(ButtonOutline, {
+                style: { marginTop: '1rem' },
+                tooltip: 'Upgrade your environment to use a persistent disk. This will require a one-time deletion of your current built-in disk, but after that your data will be stored and preserved on the persistent disk.',
+                onClick: () => this.setState({ upgradeDiskSelected: true })
+              }, ['Upgrade'])
+            ]),
+            renderBottomButtons()
+          ])
+        ])
+    }
+
+    const renderAboutPersistentDisk = () => {
+      return div({ style: styles.drawerContent }, [
+        h(TitleBar, {
+          style: styles.titleBar,
+          title: 'About persistent disk',
+          onDismiss,
+          onPrevious: () => this.setState({ viewMode: undefined })
+        }),
+        div({ style: { lineHeight: 1.5 } }, [
+          // TODO PD: this language could be cleaner, e.g. we don't abbreviate 'PD' anywhere else
```

Comment:
Just calling this out as an unresolved TODO, we should un-abbreviate PD here. Does anyone know if this was included in the full language pass? It seems like we're using 'compute' in some places where we might want to update to 'cloud environment' or 'cloud compute'

Reply:
Lou said that this was still the preferred language.